.travis.yml

```diff
@@ -10,12 +10,18 @@ matrix:
     before_install: true
     after_failure: true
     env: TRAVIS_JOB_TYPE=FORMAT
-  - name: Zabbix 3.4
-    env: ZBX_VERSION=3.4.12 ZBX_HEADERS_VERSION=3.2 TRAVIS_JOB_TYPE=TEST
-  - name: Zabbix 3.2
-    env: ZBX_VERSION=3.2.11 ZBX_HEADERS_VERSION=3.2 TRAVIS_JOB_TYPE=TEST
-  - name: Zabbix 3.0
-    env: ZBX_VERSION=3.0.19 ZBX_HEADERS_VERSION=3   TRAVIS_JOB_TYPE=TEST
+  - name: Zabbix 3.4 Modbus TCP
+    env: ZBX_VERSION=3.4.12 ZBX_HEADERS_VERSION=3.2 TRAVIS_JOB_TYPE=TEST MODBUS_HOST=172.16.238.2:5020
```

Comment:
Will it work if we change this long string into a _folded YAML scalar_ for better readability?
```yaml
env: <
  ZBX_VERSION=3.4.12
  ZBX_HEADERS_VERSION=3.2
  TRAVIS_JOB_TYPE=TEST
  MODBUS_HOST=172.16.238.2:5020
```

Reply:
done!